modules/lib/maintainers.nix

```diff
@@ -211,6 +211,12 @@
     githubId = 1472826;
     name = "Max Smolin";
   };
+  michaelCTS = {
+    email = "michael.vogel@cts.co";
+    github = "michaelCTS";
+    githubId = 132582212;
+    name = "Michael Vogel";
+  };
```

Comment:
Since you are already listed in Nixpkgs' maintainer-list.nix you don't need to be listed here as well.

Reply:
Replace all references with `pkgs.lib.maintainers.michaelCTS`